include/Gaffer/FileSystemPath.h

```diff
@@ -94,6 +97,8 @@ class GAFFER_API FileSystemPath : public Path
 		// Returns the path converted to the OS native format
 		std::string nativeString() const;
 
+		std::filesystem::path path() const;
```

Comment:
I wish I could come up with a less ambiguous name for this, since we're mixing `Gaffer::Path` and `filesystem::path` in one API here. Perhaps `stdPath()`? But then that makes for a weird C++-specific name in the Python bindings. Maybe `standardPath()`, where "standard" means "std or pathlib"? I'm not sure. If that doesn't strike you as an improvement, then just stick with what you have...

Reply:
Yeah, `path()` is not the best wording. I went with `standardPath()` in https://github.com/GafferHQ/gaffer/pull/5049/commits/0a73bc6a93e4c09b3c2b7add9edb7eff393c92fb - it's maybe not perfect but I think it's an improvement over `path()` for clarity.